static/js/add-shows/recommended-shows.js

```diff
@@ -10,6 +10,22 @@ MEDUSA.addShows.recommendedShows = function() {
     $.initAddShowById();
     $.initBlackListShowById();
     $.initRemoteShowGrid();
-    $.imgLazyLoad.update();
-    $.imgLazyLoad.handleScroll();
+
+    if ($('#rootDirs option:selected').length === 0) {
```

Comment:
Please stop duplicating functions. If a function is used on multiple pages it needs to be added to the common file. I've said this multiple times.